### PR TITLE
Example scenario for data warehousing

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/data/data-warehouse.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Data warehousing and analytics for sales and marketing
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 09/15/2018
+---
+
+# Data warehousing and analytics for sales and marketing
+
+This example scenario builds a data pipeline to integrate large amounts of data from multiple sources into a unified analytics platform in Azure. This specific scenario is based on a sales and marketing solution, but the design patterns are relevant for many industries requiring advanced analytics of large datasets such as e-commerce, retail, and healthcare.
```
Comment (MikeWasson) [inline docs/example-scenario/data/data-warehouse.md]: builds or shows?

File: docs/example-scenario/data/data-warehouse.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* [Power BI](https://powerbi.microsoft.com/pricing/) has different product options for different requirements. [Power BI Embedded](https://azure.microsoft.com/pricing/details/power-bi-embedded/) provides an Azure-based option for embedding Power BI functionality inside your applications. A Power BI Embedded instance is included in the pricing sample above.  
+
+## Next Steps
+
+* Review the [Azure reference architecture for automated enterprise BI](/azure/architecture/reference-architectures/data/enterprise-bi-adf), which includes instructions for deploying an instance of this architecture in Azure.
+* Read the [Maritz Motivation Solutions customer story][source-document]. That story describes a similar approach to managing customer data.
+* Find comprehensive architectural guidance on data pipelines, data warehousing, online analytical processing (OLAP), and big data in the [Azure Data Architecture Guide](/azure/architecture/data-guide/).
+* Learn proven practices for building Azure-based solutions in the [Azure Architecture Center](/azure/architecture/).
```
Comment (MikeWasson) [inline docs/example-scenario/data/data-warehouse.md]: Do we need this link?

File: docs/example-scenario/data/data-warehouse.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Fixed.

File: docs/example-scenario/data/data-warehouse.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+* [SQL Data Warehouse](/azure/sql-data-warehouse) is a distributed system for storing and analyzing large datasets. Its use of massive parallel processing (MPP) makes it suitable for running high-performance analytics. SQL Data Warehouse can use [PolyBase](/sql/relational-databases/polybase/polybase-guide) to rapidly load data from Blob storage.
+* [Analysis Services](/azure/analysis-services) provides a semantic model for your data. It can also increase system performance when analyzing your data. 
+* [Power BI](/power-bi) is a suite of business analytics tools to analyze data and share insights. Power BI can query a semantic model stored in Analysis Services, or it can query SQL Data Warehouse directly.
+* [Azure Active Directory (Azure AD)](/azure/active-directory) authenticates users who connect to the Analysis Services server through Power BI. Data Factory can also use Azure AD to authenticate to SQL Data Warehouse via a service principal or Managed Service Identity (MSI).
+
+### Alternatives
+
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Added.

File: docs/example-scenario/data/data-warehouse.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+* [SQL Data Warehouse](/azure/sql-data-warehouse) is a distributed system for storing and analyzing large datasets. Its use of massive parallel processing (MPP) makes it suitable for running high-performance analytics. SQL Data Warehouse can use [PolyBase](/sql/relational-databases/polybase/polybase-guide) to rapidly load data from Blob storage.
+* [Analysis Services](/azure/analysis-services) provides a semantic model for your data. It can also increase system performance when analyzing your data. 
+* [Power BI](/power-bi) is a suite of business analytics tools to analyze data and share insights. Power BI can query a semantic model stored in Analysis Services, or it can query SQL Data Warehouse directly.
+* [Azure Active Directory (Azure AD)](/azure/active-directory) authenticates users who connect to the Analysis Services server through Power BI. Data Factory can also use Azure AD to authenticate to SQL Data Warehouse via a service principal or Managed Service Identity (MSI).
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Fixed.

File: docs/example-scenario/data/data-warehouse.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+* When analysis activity is low, the company can [scale SQL Data Warehouse on demand](/azure/sql-data-warehouse/sql-data-warehouse-manage-compute-overview), reducing or even pausing compute to lower costs.
+* Azure Analysis Services can be [scaled out](/azure/analysis-services/analysis-services-scale-out) to reduce response times during high query workloads. You can also separate processing from the query pool, so that client queries aren't slowed down by processing operations. 
+* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+* The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication.
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Fixed.

File: docs/example-scenario/data/data-warehouse.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Removed.

**Pull request merged**